ui/app/templates/permission_scheduled_changes.html

```diff
@@ -42,13 +42,13 @@
     <h3 class="panel-title">
       <div style="float: right">
         <i ng-if=" scheduled_changes_rules_count && $first && (currentPage == 1)">Current</i>
-        <button class="btn btn-xs btn-primary" ng-click="signoff(sc)"
-                ng-show="! isEmpty(sc['required_signoffs']) && ! sc['signoffs'].hasOwnProperty(current_user)">
-            Signoff as...
+        <button ng-show="!scheduled_changes_rules_count && (! isEmpty(sc['required_signoffs']) &&
```

Comment:
I think you've got a copy/paste error here and on the releases page - scheduled_changes_rules_count doesn't exist. See below about changing this to use a different variable, too.

Reply:
@mozbhearsum The copy/paste error, yes. I have realized that permissions have no history. I have also changed to detect page using `sc.sc_id` please check if its okay now.

Reply:
Did you forget to push an update? This seems to be unchanged since the last time I looked at it.